modules/graphql/server-ts/createApolloServer.ts

```diff
@@ -0,0 +1,37 @@
+import { ApolloServer, AuthenticationError, ApolloError } from 'apollo-server-express';
+import { formatResponse } from 'apollo-logger';
+import 'isomorphic-fetch';
+import { log } from '@gqlapp/core-common';
+import { GraphQLConfigShape } from '@gqlapp/graphql-server-ts';
+
+import settings from '../../../settings';
+
+const {
+  engine: { apiKey },
+  app: { logging }
+} = settings;
+
+export default ({ schema, createGraphQLContext }: GraphQLConfigShape) => {
+  const context = async ({ req, res }: { req: any; res: any }) => ({
+    ...(await createGraphQLContext(req, res)),
+    req,
+    res
+  });
+
+  const formatError = (error: ApolloError) =>
+    error.message === 'Not Authenticated!' ? new AuthenticationError(error.message) : error;
+
+  const formatResponseCustom = (response: any, options: { [key: string]: any }) =>
```

Comment:
If you want to stress that this is a custom function, its name should be `customFormatResponse`. But still it will be the bad name, because it does not reveal what this function does. A better name here is `formatAndLogResponse`

Reply:
Renamed.